.github/policies/eventResponder.yml

```diff
@@ -17,6 +17,16 @@ configuration:
         then:
           - addLabel:
               label: "Needs: Triage :mag:"
+          - addReply:
+              reply: |
+                > [!IMPORTANT]
+                > **The "Needs: Triage :mag:" label must be removed as part of the triage process (when the issue/PR is first responded to) !**
```

Comment:
I would recommend you use the same wording you used in the [AVM PR](https://github.com/Azure/Azure-Verified-Modules/pull/763/files): `**The "Needs: Triage :mag:" label must be removed once the triage process is complete !**`.

The 'first responded to' is (imho) way to strict. Anybody can 'first respond' to a PR - that doesn't mean it is triaged. I'd argue the only people that should triage are the owners (and at most maintainers). 

Good thing you already did have a 'softer' wording created ;)